### PR TITLE
feat: update en and de translations

### PR DESCRIPTION
**Pull request by misiekhardcore**

### CODE REVIEW

File: whats-new/whatsnew_en.md [hunk 1]
```diff
@@ -1,22 +1,7 @@
...
-
 ## Carbon Design migration (All distributions)
+
 We are in the process of migrating Camunda Optimize UI to use the Carbon Design System. Our primary focus is to bring the Optimize UI in line with other Camunda products, providing a consistent and seamless experience across the entire ecosystem. With this migration, we will also adhere to accessibility best practices, ensuring that our application is usable by a broader range of users.
 
 The migration to the Carbon Design System for Camunda Optimize is an ongoing effort. We have successfully migrated modals, tables, and popovers, ensuring these components seamlessly integrate into the new design. While the functionality you love remains intact, you can expect a refreshing new UI that aligns with the Carbon Design System.
+
+The updated Task Analysis now includes a revamped outliers table, offering enhanced insights into your processes. This table provides detailed information about outlier process instances, including the number of instances and related variables. For even more in-depth details, you can explore the modal, which showcases outliers on the duration chart and in the table.
```
Comment (OmranAbazid) [inline whats-new/whatsnew_en.md:7]: I guess this section needs a title.

File: whats-new/whatsnew_de.md [hunk 2]
```diff
@@ -1,22 +1,7 @@
...
-
-## Beschreibung für Reports und Dashboards
-Eine Beschreibung, wie Daten verstanden werden sollen, sowie eine Beschreibung des Kontexts der Daten verbessert deren Verständlichkeit beträchtlich. Mit dem Release 3.10 wurden Text Reports eingeführt. Diese können auf Dashboards als Kacheln neben Daten Reports positioniert werden. Mit diesem Release ist es jetzt möglich Beschreibungen direkt für einen Report oder Dashboard zu definieren.
-
 ## Carbon Design System Migration
+
 Aktuell befinden wir uns in einer Migration der aktuellen Optimize Oberflächen auf das Carbon Design System. Ziel ist es die Optimize Oberflächen mit anderen Camunda Produkten zu vereinheitlichen. Mit dem Carbon Design System gehen wir einen großen Schritt in Richtung der Umsetzung eines Barrierefreien Zugangs, um unsere Anwendungen einem breiten Spektrum von Nutzern zugänglich zu machen.
 
 Die Migration erfolgt schrittweise mit kommenden Releases von Optimize. Aktuell wurden Modal-Dialoge, Tabellen und Pop-Overs erfolgreich migriert. Die bekannten Funktionen sind weiter in gewohnter Weise verfügbar, ab sofort jedoch im frischen Carbon Design.
+
```
Comment (OmranAbazid) [inline whats-new/whatsnew_de.md:6]: same here

**Review by OmranAbazid — COMMENTED** [review]
Did Tobias reviewed that what's new changes? he is usually the person responsible for writing the what's new content

File: whats-new/whatsnew_en.md (hunk 1 above)
Comment (misiekhardcore) [reply whats-new/whatsnew_en.md:7]: Yes, I posted this on the task analysis epic channel and he approved it

**Pull request merged**